src/Styles/StyledComponents.js

```diff
@@ -43,6 +50,65 @@ export const NavLinks = styled.a`
   }
 `;
 
+export const MenuIcon = styled.a`
+  align-self: center;
+  margin-right: 1em;
+  text-decoration: none;
+  :active {
+    color: orange;
+  }
+  @media (min-width: 480px) {
+    display: none;
+  }
+`;
+
+export const MobileMenu = styled.div`
+  height: 250px;
+  box-shadow: 2px 2px 8px 2px rgba(0, 0, 0, 0.5);
+  background: ${darkerGrey};
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 200;
+  transform: translateY(-100%);
+  transition: transform 0.3s ease-out;
+
+  ${props =>
```

Comment:
how common is this to have css mixed with javascript? it looks like an anti-pattern to me.

Reply:
Should this change happen on the component itself? ie: https://stackoverflow.com/questions/22291913/how-to-change-reactjs-styles-dynamically

Reply:
This was a solution I found to triggering this animation a while back using styled-components. Seems to be common on for styled-components. https://www.styled-components.com/docs/basics